app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.players.elves import *
+from app.players.dwarves import *
+
```

Comment:
You don't need to import like this, it's bad practice. And actually, you may not use import in main.py